pkg/util/fs.go

```diff
@@ -77,39 +77,77 @@ func (h *fs) Exists(file string) bool {
 	return err == nil
 }
 
-// Copy copies a set of files from sourcePath to targetPath
-func (h *fs) Copy(sourcePath string, targetPath string) error {
-	if _, err := os.Stat(sourcePath); err != nil {
+// Copy copies the source file to a destination directory.
+// If the destination is a filename, the file will be replaced.
+// If the source is directory, the directory itself and also the content of the
+// source directory will be copied into a target directory.
+// If the destination is a directory an error will be returned as the
+// destination has to be full path to a file.
```

Comment:
the previous two statements are contradictory.  either the content will be copied into the target directory, or the target cannot be a directory, which is it?

and what is the behavior when the target does not exist?